support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx

```diff
@@ -146,119 +170,165 @@ export function SupporterPlusLandingPage({
 		}
 	}, [paymentComplete]);
 
-	return (
-		<PageScaffold
-			id="supporter-plus-landing"
-			header={
-				<>
-					<Header>
-						<Hide from="desktop">
-							<CountrySwitcherContainer>
-								<CountryGroupSwitcher {...countrySwitcherProps} />
-							</CountrySwitcherContainer>
-						</Hide>
-					</Header>
-					<Nav {...countrySwitcherProps} />
-				</>
-			}
-			footer={
-				<FooterWithContents>
-					<FooterLinks></FooterLinks>
-				</FooterWithContents>
+	const [fullFormDisplayed, setFullFormDisplayed] = useState(
+		!optimisedMobileLayout2,
+	);
+
+	function onStickyButtonClick() {
```

Comment:
I think we need to add a tracking call on this button using the `sendTrackingEventsOnClick` function so we measure click through rate when evaluating the test...

```
sendTrackingEventsOnClick({
    id: 'sticky_cta',
    componentType: 'ACQUISITIONS_BUTTON',
})();
```